openedx/features/edly/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.contrib.sites.models import Site
+from django.core.validators import RegexValidator
+
+
+class Organization(models.Model):
+    alphanumeric = RegexValidator(r'^[0-9a-zA-Z]*$', 'Only alphanumeric characters are allowed.')
+
+    name = models.CharField(max_length=100, unique=True, validators=[alphanumeric])
+    slug = models.CharField(max_length=100, unique=True, validators=[alphanumeric])
+
+    def __repr__(self):
+        return self.name
+
+
+class SubOrganization(models.Model):
+    alphanumeric = RegexValidator(r'^[0-9a-zA-Z]*$', 'Only alphanumeric characters are allowed.')
+
+    name = models.CharField(max_length=100, unique=True, validators=[alphanumeric])
+    slug = models.CharField(max_length=100, unique=True, validators=[alphanumeric])
+    organization = models.ForeignKey(Organization, null=False, blank=False)
+    lms_site = models.ForeignKey(Site, related_name='lms_site')
```

Comment:
This should be `OneToOneField` field and same for `studio_site`

Reply:
Done